packages/logbook/meta.yaml

```diff
@@ -12,11 +12,6 @@ requirements:
     - setuptools
   run:
     - ssl
-
-build:
-  script: |
-    python -c 'from Cython.Build.Cythonize import main; main(["./logbook/_speedups.pyx"])'
```

Comment:
logbook now does this in their setup.py, so this line is not required anymore.

https://github.com/getlogbook/logbook/blob/1.7.0.post0/setup.py